aioxmpp/e2etest/provision.py

```diff
@@ -725,3 +728,100 @@ def _make_client(self, logger):
             override_peer=override_peer,
             logger=logger,
         )
+
+
+class StaticPasswordProvisioner(_AutoConfiguredProvisioner):
+    """
+    This provisioner expects a list of username/password pairs to authenticate
+    against the tested server.
+
+    This is for use with servers which support neither SASL ANONYMOUS nor
+    a ``mod_auth_any`` equivalent.
+
+    The configuration of this provisioner is slightly unwieldly since we do
+    not want to add a dependency to a more sane configuration file format. Here
+    is an example on how to configure a provisioner with three accounts:
```

Comment:
Three? The example shows two.

Reply:
That’s because I got lazy after the first two. Thanks.